classes/openrc-image.bbclass

```diff
@@ -50,6 +50,34 @@ openrc_add_services() {
     done
 }
 
+openrc_disable_services() {
+    for pair in ${OPENRC_DISABLED_SERVICES}; do
```

Comment:
This makes sense as a companion to `OPENRC_SERVICES`, but please add a default value with documentation at the top of this file similar to `OPENRC_SERVICES`

Reply:
Ah yes, will do.